packages/react-composites/src/composites/CallComposite/components/CallControls.tsx

```diff
@@ -247,8 +253,11 @@ export const CallControls = (props: CallControlsProps): JSX.Element => {
       data-ui-id="call-composite-microphone-button"
       {...microphoneButtonProps}
       showLabel={!compactMode}
+      // xkcd: style this correctly to increase touch targets.
```

Comment:
fixup todo

Reply:
Done in a PR lower in the stack (the actual styling one...)